app.py

```diff
@@ -182,6 +182,25 @@ def login_():
     def logout_():
         return logout()
 
+    @app.route('/push-log', methods=['POST'])
+    def push_log():
+        if 'level' not in request.json or 'message' not in request.json:
+            raise ValueError('Request body missing on or more mandatory fields ["message", "level"]')
+
+        _json = request.json.get
+        level, message, extra = _json('level'), _json('message'), _json('extra')
+        if not (extra is None or type(extra) is dict):
+            raise ValueError('Extra param must be a valid json object')
+
+        logging_fun = getattr(logger, level.lower(), None)
+
+        if logging_fun is None:
+            raise ValueError('Invalid logging level requested')
```

Comment:
So I thought about it but I preferred to not use a default level since if by mistake a level was not specified for a "debug" message, then it would end up in the log with info that should probably not be seen